layout/templates/mobileMapUI.mustache

```diff
@@ -5,7 +5,7 @@
         {{{selectArrows}}}
       </div>
       <select class="view-select">
-        <option value="United States">United States - Water Use</option>
+        <option value="USA">United States - Water Use</option>
```

Comment:
It doesn't make sense to me to have ` - Water Use` for United States only, not for states. I could see either adding ` - Water Use` to every state name or leaving it off of all of them. I'd advocate for leaving it off - I think that info is better suited for the header (where it already is) and/or the title or subtitle of the legend.